uvcombine/scale_factor.py

```diff
@@ -0,0 +1,227 @@
+
+"""
+Routines for determining the scale factor between single dish and
+interferometric data.
+
+
+"""
+
+from astropy import units as u
+from astropy.utils.console import ProgressBar
+import numpy as np
+from astropy import stats as astrostats
+from scipy import stats
+
+from .uvcombine import feather_compare
+
+
+def find_effSDbeam(hires, lores,
+                   LAS,
+                   lowresfwhms,
+                   beam_divide_lores=True,
+                   highpassfilterSD=False,
+                   min_beam_fraction=0.1,
+                   alpha=0.85,
+                   verbose=False):
+    '''
+    Find the optimal FWHM of the SD data by minimizing the relation between
+    the ratios in the overlapping region
+
+    See Sec. 3.2.1 in Stanimirovic (1999)
+    https://ui.adsabs.harvard.edu/#abs/1999PhDT........21S/abstract
+
```

Comment:
could you add parameter definitions here?  especially, what is `lowresfwhms` being used for?